hasher-matcher-actioner/hmalib/lambdas/api/submit.py

```diff
@@ -206,35 +216,22 @@ def from_url(
         """
         Submission via a url to content. Current behavior copies content into the system's s3 bucket.
         """
-        fileName = request.content_id
+        content_id = request.content_id
         url = request.content_bytes_url_or_file_type
-        response = requests.get(url)
-        # TODO better checks that the URL actually worked...
-        if response and response.content:
-            # TODO a whole bunch more validation and error checking...
 
-            # Again, We want to record the submission before triggering and processing on
-            # the content itself therefore we write to dynamo before s3
-            record_content_submission(dynamodb_table, image_folder_key, request)
+        # Again, We want to record the submission before triggering and processing on
+        # the content itself therefore we write to dynamo before s3
+        record_content_submission(dynamodb_table, image_folder_key, request)
 
-            # Right now this makes a local copy in s3 but future changes to
-            # pdq_hasher should allow us to avoid storing to our own s3 bucket
-            # (or possibly give the api/user the option)
-            s3_client.put_object(
-                Body=response.content,
-                Bucket=image_bucket_key,
-                Key=f"{image_folder_key}{fileName}",
-            )
+        url_submission_message = URLImageSubmissionMessage(content_id, t.cast(str, url))
+        _get_sns_client().publish(
+            TopicArn=images_topic_arn,
+            Message=json.dumps(url_submission_message.to_sqs_message()),
+        )
 
-            return SubmitContentResponse(
-                content_id=request.content_id, submit_successful=True
-            )
-        else:
-            bottle.response.status = 400
-            return SubmitContentError(
-                content_id=request.content_id,
-                message="url submitted could not be read from",
-            )
+        return SubmitContentResponse(
+            content_id=request.content_id, submit_successful=True
```

Comment:
The hasher doesn't know about `IMAGE_FOLDER_KEY` therefore the way that class handles writing `PipelinePDQHashRecord` for s3 vs URL submissions ~breaks. 

Where this was 'fixed' (or goes wrong depending on perspectives) can be seen here: https://github.com/facebook/ThreatExchange/blob/master/hasher-matcher-actioner/hmalib/lambdas/api/submit.py#L124

Right now the system (and the UI) assumes the content_id stored in the dyanmodb is prefixed with `IMAGE_FOLDER_KEY`

I think we do want to change this but the move fast for now change is to add the prefix here.